### ISSUE TITLE
NVDA/Narrator is not announcing the copied code successfully message : A11y_WCP URLs - ONNX Runtime_Home_Machine learning_Screen reader.

### ISSUE THREAD

**Issue by PadminiGariki** [microsoft/onnxruntime#20120]
### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopWeb; #SH_ONNX Runtime & Ecosystem_Web_Mar2024; #ONNX Runtime & Ecosystem; #WCP; #Win11; #Benchmark; #FTP; #P3_WCP; #ONNX Runtime; #ChromiumEdge; #STP; #Screen reader; #NVDA; #Narrator; #WCAG4.1.3; #Status  Messages;

### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.53 (Official build) (64-bit)
OS: Windows 11 version 23H2
Screen reader: Narrator, NVDA: version 2023.3.3.

### Steps to Reproduce

1.Turn on the NVDA/Narrator.
2.  Hit the URL "https://onnxruntime.ai/".
3. "ONNX Runtime" home page is appeared.
4. Tab till "Machine learning" section.
5. Tab till "Pip install onnx runtime" Crop button and press enter.
6. Now check that NVDA/Narrator is not announcing the copied code successfully message.

### Actual Results:
NVDA/Narrator is not announcing the copied code successfully message.

### Expected Results:
NVDA/Narrator Should announce the copied code successfully message.

### User Impact:
Screen reader user will face issue. There not able know about copied code successfully message information. If NVDA/Narrator is not announcing.

### Attachment:

![Screenshot 2024-03-28 115858](https://github.com/microsoft/onnxruntime/assets/142479541/3a763f20-39d3-4d85-9a9b-1a03c97e3b05)

**Comment by v-jagansai** [comment]
Verified the issue in below environment.
Application Name: WCP URLs - ONNX Runtime
URl: https://onnxruntime.ai/
Edge version: 123.0.2420.97 (Official build) (64-bit)
OS: Windows 11 version 23H2

Issue is fixed. Hence the bug is closed.
#Regressed:04-18-24; #Closed;

For reference attachment is added.
[20120_Closed.zip](https://github.com/microsoft/onnxruntime/files/15021316/20120_Closed.zip)
